google-cloud-tools-plugin/testData/project/java.yaml

```diff
@@ -0,0 +1,15 @@
+runtime: java
+service: javaService
+env: flex
+
+handlers:
+- url: /.*
+  script: this field is required, but ignored
+  secure: always  # Require HTTPS
+
+runtime_config:  # Optional
+  jdk: openjdk8
+  server: jetty9
+
+manual_scaling:
+  instances: 1
```

Comment:
missing new line. just curious, is this the yaml generated by gen-config?  I'm asking because there's an arbitrary amount of detail/noise in this test data that we don't really care about.